modules/search/instagram.py

```diff
@@ -19,60 +19,87 @@
 
 meta = {
 	'name': 'Instagram Search',
-	'author': 'Aman Singh',
+	'author': 'Rishabh Jain, Aman Singh',
 	'version': '0.2',
 	'description': 'Search your query in the Instagram and show the results.',
-	'sources': ('google', 'carrot2', 'bing', 'yippy', 'yahoo', 'millionshort', 'qwant', 'duckduckgo'),
+	'sources': ('google', 'carrot2', 'bing', 'yippy', 'yahoo', 'millionshort', 'qwant', 'duckduckgo','instagram'),
 	'options': (
 		('query', None, True, 'Query string', '-q', 'store', str),
 		('limit', 1, False, 'Search limit(number of pages, default=1)', '-l', 'store', int),
 		('count', 50, False, 'Number of links per page(min=10, max=100, default=50)', '-c', 'store', int),
+		('engine', 'google,instagram', False, 'Engine names for search(default=google, instagram)', '-e', 'store', str),
+		('session_id', None, False, 'Session_id for more details results (default="")', '-s', 'store', str),
```

Comment:
What is session id?

Reply:
for extracting more data of an account, I use session_id of any account use created when login 
using an account's session_id results will be related to that account (this is the way Instagram works)
and using session_id you will never get an error in response

Reply:
What is session id? This is a cookie?